toolchains/clang/clang_toolchain.bzl

```diff
@@ -78,6 +78,10 @@ def _get_additional_system_include_paths(ctx):
 
 def _clang_toolchain_config_info_impl(ctx):
     tool_paths = [
+        tool_path(
+            name = "cpp",
+            path = "clang_wrappers/{os}/cpp",
+        ),
```

Comment:
This already exists a few stanzas down from here.

Reply:
Thanks, I'll remove it.